Src/AutoFixture.NUnit3.UnitTest/PilotTest.cs

```diff
@@ -0,0 +1,14 @@
+﻿using NUnit.Framework;
+
+namespace Ploeh.AutoFixture.NUnit3.UnitTest
+{
+    [TestFixture]
+    public class PilotTest
+    {
+        [Test]
+        public void AssertTrue()
+        {
+            Assert.IsTrue(true);
```

Comment:
Is this test supposed to be discovered? I've changed it `false` to see if it fails, run `build-release.sh` and it succeeded.